httpi.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'rack'
   s.add_dependency 'socksify'
+  s.add_dependency 'multipart-post'
```

Comment:
Please change to `s.add_development_dependency`, this gem should be "optional" to httpi.